app/Activity.php

```diff
@@ -14,4 +14,35 @@ public function space() {
     public function user() {
         return $this->belongsTo(User::class);
     }
+
+    public function getFormattedCreatedAtAttribute() {
+        $date1 = strtotime($this->created_at);
+        $date2 = strtotime('now');
```

Comment:
Either elaborate in these variable names in a comment, or name them so that an outsider knows what they're for.